.devcontainer/run-livebook-stable/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+// This devcontainer is for pulling the latest stable Livebook image from Docker Hub.
+
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/blob/v0.231.6/container-templates/image/.devcontainer/devcontainer.json
+{
+  
+  // Update the 'image' property with your Docker image name.
+  "image": "livebook/livebook",
```

Comment:
We also build `livebook/livebook:edge` that matches the main branch, so maybe use that for the development version to skip the build?